Remotion/Core/Core/Reflection/TypeExtensions.cs

```diff
@@ -113,7 +113,7 @@ private static bool CanAscribeInternal (Type type, Type ascribeeType)
         return ascribeeType.IsAssignableFrom (type);
 
       Type ascribeeGenericTypeDefinition = ascribeeType.GetGenericTypeDefinition();
-      for (Type currentType = type; currentType != null; currentType = currentType.BaseType)
+      for (Type currentType = type; currentType != null; currentType = currentType.BaseType!)
```

Comment:
Is this how you do for loops with nullable? I would have thought the declaration is nullable and then due to the loop condition / flow analysis, it would be assumed as not-null.

Reply:
I guess search & replace got out of hand with this one, my bad.